lib/src/store.dart

```diff
@@ -85,8 +85,10 @@ class Store extends Disposable {
       throw new StateError('Store has been disposed');
     }
 
-    return _stream.listen(onData,
```

Comment:
I don't recall the exact discussions that lead up to the final decision, but here is the PR: https://github.com/Workiva/w_flux/pull/14

Despite me making the PR I don't think I was a huge fan of the decision at the time. IIRC it was mostly concern around API pollution. We figured it would be better to be explicit about what we expose, and proxy and methods to stream that we want to support. I agree with that stance now